src/NuGet.Clients/NuGet.CommandLine/Common/ProjectHelper.cs

```diff
@@ -16,6 +16,7 @@ public static class ProjectHelper
             ".jsproj",
             ".wixproj",
             ".nuproj",
+            ".nfproj",
```

Comment:
Do you want this in https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets?

NuGet has PackageReference & packages.config and the NuGet.targets are primarily PackageReference support, but they are able to do packages.config projects in certain scenarios.

You can read more about that in https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#restoring-packagereference-and-packagesconfig-with-msbuild

I'd also encourage having the TFM discussion on the dotnet designs repo.

Reply:
> I'd also encourage having the TFM discussion on the dotnet designs repo.

Already had a meeting about this the other week. A proposal is about to be posted there for dicussion. 😉

Reply:
I wanted to repeat the question, 
> Do you want this in https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets?

nfproj currently supports packages.config only. 

MSBuild targets support a version of packages.config restore as well.

You'd have to add a test for msbuild packages.config restore in https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Core.FuncTests/Msbuild.Integration.Test/MsbuildRestoreTaskTests.cs. 

https://docs.microsoft.com/en-us/nuget/consume-packages/package-restore#restore-using-msbuild

Reply:
OK. Adding that.

Reply:
On a second thought: that test won't work. Our project system doesn't use the .NET targets that include the restore.

Reply:
And answering the original question: I don't think we need that with the current project system. So, the answer is no.

Reply:
sounds good, resolving this thread.